src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -289,10 +289,17 @@ def make_dataframe_simulation_data(
         nb_real: number of realizations
 
     """
-    iteration = sorted(
-        [int(rel_iter.replace("/", "").split("-")[-1]) for rel_iter in glob.glob(path)]
-    )[-1]
-    runpath_list = glob.glob(path[::-1].replace("*", str(iteration)[::-1], 1)[::-1])
+    if "pred" in path:
```

Comment:
Is that always true?

Reply:
> > @wouterjdb: I changed the well_connections property of FlowData to a getter method as I want to pass it the argument of perforation_handeling_strategy and you can't pass arguments to a property. Pylint doesn't seem to like this however:
> > `Parameters differ from overridden 'get_well_connections' methodpylint(arguments-differ)`
> > Any ideas on how to fix this? Not so familiar with properties.
> 
> The method you implement has different arguments from the definition of the abstract method. I.e., you did not implement what is expected according to the definition of the abstract method. This is why pylint complains.
> 
> So, update the abstract method to reflect that the perforation strategy is part of the method.

Ah of course!

Reply:
You are right. Didn't realize runpath was also an optional argument in the config. This is fixed by passing an additional keyword to the analysis workflow defining the mode in which flownet is run: pred or ahm (commit: [feda7a7](https://github.com/equinor/flownet/pull/349/commits/feda7a77d5b15330043348da84d90341f7623346))